Add documentation for new upnp binary sensor

ehendrix23 opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Add documentation for new upnp binary sensor


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54489
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ehendrix23 committed:
Add documentation for new upnp binary sensor

ehendrix23 committed:
Small update

frenck commented on source/_integrations/upnp.markdown:
```diff
@@ -12,13 +13,15 @@ ha_codeowners:
 
 There is currently support for the following device types within Home Assistant:
 
+- **Binary Sensor** - Router connected to WAN, attributes include actual status text, WAP IP, and uptime of router.
```
How can a binary sensor include text?

ehendrix23 replied:
Was included as attributes. Removed now and will be added through other PR as new sensors. Documentation updated to reflect current PR.

ehendrix23 committed:
Removed attributes for binary sensor

Attributes are removed from the binary sensor and will be added through another PR as sensors. Hence updating documentation.

frenck reviewed (approved):
✅ Approved. Can be merged as soon as the parent PR gets merged.
